util/processadorProtocolo.py

```diff
@@ -56,7 +57,18 @@ def processaConexao(sConexao):
             ordemComando += 1
             sConexao.send('354 Enter mail, end with ".". on a line by itself'.encode('UTF8'))
             # SECTION - receber email
-            _ = sConexao.recv(TAM_BUFFER_RECV) # STUB
+            mensagemBytes = sConexao.recv(TAM_BUFFER_RECV)
+            mensagem = mensagemBytes.decode('UTF8')
+            nome_caixa_entrada = emailDestinatario.split("@")[0]+".txt"
+            #_ = sConexao.recv(TAM_BUFFER_RECV)
+            with open(nome_caixa_entrada, "w") as caixaDeEntrada:
```

Comment:
deve abrir para escrever no fim do arquivo
```suggestion
            with open(nome_caixa_entrada, "a") as caixaDeEntrada:
```